app/controllers/api/v1/meetings_controller.rb

```diff
@@ -0,0 +1,71 @@
+class Api::V1::MeetingsController < ApplicationController
+  before_action :set_users, only: [:create]
+  before_action :existing_partner_check, only: [:create]
+  before_action :set_meeting, only: [:update, :destroy]
```

Comment:
Clever, I always wonder about the value of callbacks vs facades, definitely an interesting topic